ten-leetcode-problems/MissMatch.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ten_leetcode_problems
+{
+    internal static class MissMatch
+    {
+        internal static int[] Find(int[] nums)
+        {
+            int[] result = new int[2];
+            Array.Sort(nums);
```

Comment:
time complexity of this problem is O(nlogn), maybe we can use a dictionary and aim for a linear solution?